docs: add edge CLI command updates (#2703)
spectrocloud/librarium: docs/docs-content/clusters/edge/edgeforge-workflow/build-content-bundle.md

    --project-id <PROJECT_ID> \
    --cluster-profile-ids <CLUSTER_PROFILE_ID1,CLUSTER_PROFILE_ID2...> \
    --palette-endpoint <Palette API Endpoint> \
    --outfile <bundle-name> \
    --cred-file-path <file-path> \
    --include-palette-content \
   ```

   | Flag                        | Description                                                                                                    |
   | --------------------------- | -------------------------------------------------------------------------------------------------------------- |
   | `--api-key`                 | Your Palette API key.                                                                                          |
   | `--cluster-profile-ids`     | Comma-separated list of cluster profile IDs to download content for.                                           |
   | `--cred-file-path`          | Path to the JSON file storing registry credentials if you are using a private registry.                        |

>    | `--include-palette-content` | Whether to include necessary content for Palette itself. Required if for airgap installations.                 |

Reviewer: ```suggestion
   | `--include-palette-content` | Whether to include content necessary for Palette itself. Required for airgap installations.                 |
```

Reviewer: I was confused with what necessary refers to, so moved around a bit 👁️

>    | `--outfile`                 | Name of your content bundle. The final file name looks like the following: `core-<bundle-name>-random-string`. |

Reviewer: ```suggestion
   | `--outfile`                 | Name of your content bundle. The final file name should have the following pattern: `core-<bundle-name>-random-string`. |
```

   | `--palette-endpoint`        | API endpoint for your Palette instance.                                                                        |
   | `--project-id`              | The ID of your Palette project.                                                                                |

   The result is a content bundle that you can use to preload into your installer. For more information, refer to
   [Build Edge Artifacts with Content Bundle](./palette-canvos/build-artifacts.md) or
   [Build Installer ISO](./palette-canvos/build-installer-iso.md). Our Tech Preview feature
   [local UI](../local-ui/local-ui.md) also allows you to upload content bundles to a disconnected Edge deployment.

   Alternatively, you can use the ISO version of the content bundle and transfer it to a USB drive to be used separately
   at the time of Edge host installation using the `-iso` flag in your build command. Doing so will override the file

>    extension you provide using the `--outfile` flag.

Reviewer: Should this be a tip or something? Feels important 🤔
